doc/classes/ScriptEditorBase.xml

```diff
@@ -18,6 +18,13 @@
 				Adds a [EditorSyntaxHighlighter] to the open script.
 			</description>
 		</method>
+		<method name="get_base_editor" qualifiers="const">
+			<return type="Control">
+			</return>
+			<description>
+				Return either [CodeEdit] or [GraphEdit] of this editor.
```

Comment:
I think this description can be a bit more verbose about when can you expect which.

```suggestion
				Returns the underlying [Control] used for editing scripts. This can be either [CodeEdit] (for text scripts) or [GraphEdit] (for visual scripts).
```

Reply:
@pycbouh Can I commit as is (click commit change button), or need to to rebase and force push?

Reply:
On this main repo we require all PRs to be squashed. You don't necessarily need to rebase against the newest upstream branch, but you need to squash your commits anyway and force push. I'm not sure if you can commit suggestion via GitHub and then squash it externally.

I see you've already done it, but just to answer your question 🙂

Reply:
Well, I actually committed suggest then rebased at local.
Thank you for clarify this repo policy. (Also that pre-commit hook too, that was the first time I hear about git hook.)